### PR TITLE
pdpb: add fields in StoreStats to record cpu usages and disk io rates

### PR DESCRIPTION
**Pull request by jiyingtk**
Modify the pdpb.proto, add fields in StoreStats to record cpu usages and disk io rates.
This PR can help PD to know the TiKV's resource usages of both cpu and disk.

### CODE REVIEW

File: proto/pdpb.proto [hunk 1]
```diff
@@ -440,6 +447,12 @@ message StoreStats {
     uint64 keys_read = 14;
     // Actually reported time interval
     TimeInterval interval = 15;
+    // CPU usages of threads in the store
+    map<string, RollingStats> cpu_usages = 16;
```
Comment (siddontang) [inline proto/pdpb.proto]: I think it is better to use total statistics of all threads for PD. 

For thread's statistics, we can send them to Prometheus.

Comment (jiyingtk) [reply proto/pdpb.proto]: Maybe the total statistics of all threads is enough for PD now, but actually, I want to divide all threads into several categories, such as grpc, rocksdb, raft, etc., so PD can use this these diverse information to make intelligent decisions in the future.

**Comment by jiyingtk** [comment]
PTAL @rleungx @nolouch

File: proto/pdpb.proto [hunk 2]
```diff
@@ -441,6 +446,14 @@ message StoreStats {
     uint64 keys_read = 14;
     // Actually reported time interval
     TimeInterval interval = 15;
+    // Threads' CPU usages in the store
+    repeated RecordPair cpu_usages = 16;
+    // Threads' read disk I/O rates in the store
+    repeated RecordPair read_io_rates = 17;
+    // Threads' write disk I/O rates in the store
+    repeated RecordPair write_io_rates = 18;
+    // Operations' latencies in the store
```
Comment (rleungx) [inline proto/pdpb.proto]: Do we need to temporarily remove this? Once we have the corresponding PR for it, we can add it back.

**Comment by hicqu** [comment]
PTAL @nolouch @rleungx

**Pull request merged**